etc/cnid_dbd/cnid_metad.c

```diff
@@ -33,20 +33,20 @@
 #include <unistd.h>
 #undef __USE_GNU
 
-#include <stdlib.h>
-#include <sys/param.h>
+// #define _XPG4_2 1
```

Comment:
What does this commented out macro mean?

Reply:
Refers to ancient SunOS version so removed